src/main/resources/common_at.cfg

```diff
@@ -135,6 +135,7 @@ public net.minecraft.entity.item.EntityFallingBlock field_145816_i # fallHurtAmo
 public net.minecraft.entity.item.EntityFallingBlock field_145815_h # fallHurtMax
 public net.minecraft.entity.item.EntityFallingBlock field_145808_f # canSetAsBlock
 public net.minecraft.entity.item.EntityFallingBlock field_145809_g # hurtEntities
+public net.minecraft.entity.EntityLiving field_82179_bU # persistenceRequired
```

Comment:
Move this **above** `public net.minecraft.entity.EntityLiving func_175446_cd()Z # isAIDisabled`